Src/FluentAssertions/Execution/DefaultAssertionStrategy.cs

```diff
@@ -1,43 +1,62 @@
-﻿using System.Collections.Generic;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
 
 using FluentAssertions.Common;
 
 namespace FluentAssertions.Execution
 {
-    internal class DefaultAssertionStrategy : IAssertionStrategy
+#if NET45
+    [Serializable]
+#endif
+    internal class CollectingAssertionStrategy : IAssertionStrategy
```

Comment:
`DefaultAssertionStrategy` is overwritten with the code of `CollectingAssertionStrategy`

Reply:
Fixed